ADA/ada.qmd

```diff
@@ -21,55 +21,124 @@ It brings together:
+
+* Easier collaboration within workspaces
+
+* Better transparency of work being undertaken in the department by making use of the ADA shared reports area
 
 ---
 
-## Walkthrough a simple analysis
+### Making the most of Databricks 
+
+In order to take advantage of the benefits listed above, we recommend that you and your team:
+
+* Arrange data migration and access to Databricks. Onboarding is currently voluntary, so it's a great opportunity to move early and have more time to get familiar with the system and undertake training. Get in touch with [the ADA team](mailto:ADAPTTeam@education.gov.uk) to discuss onboarding with them. 
 
-See the [ADA support page on using R, python and SQL in Databricks](https://analytical-data-access.azurewebsites.net/get-support/using-r-python-and-sql-in-databricks)
+* Take part in Databricks training for R and/or SQL. R training is currently offered by the ADA team on a monthly basis and SQL training is in the process of being developed. Upcoming training is advertised on the [ADA user group Teams channel](https://teams.microsoft.com/l/team/19%3AGQW24K3V6M873QzUwpBIuyf8KZCxYByRawVsKQvpe-81%40thread.tacv2/conversations?groupId=c70bb59d-c5df-4af5-849f-36196ed14ba3&tenantId=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9).
 
+* Be proactive in determining whether any changes will be necessary for your existing code and set time aside to make these changes - see the guidance in the [what this means for existing code](/ADA/ada.html#what-this-means-for-existing-code) section below. 
+
+::: {.callout-note}
+The earlier your data is migrated, the more time you will have to dual run code from your existing methodology against code run on the Databricks platform. We encourage all analysts to engage with training and ask the ADA team about any questions or concerns as soon as possible so that you are prepared for any upcoming changes.
+:::
+
+For teams that have invested in RAP principles and already have existing code pipelines, it is possible to take advantage of cloud computing and use of data in the Databricks platform with minimal changes. This is covered in more detail in the [what this means for existing code](/ADA/ada.html#what-this-means-for-existing-code) section below. 
+
+::: {.callout-important}
+Once your team's data has been migrated, you will **not** lose access to your existing data sources, so you will be able to dual run your code using data from SSMS and data from the Databricks SQL warehouse to ensure that everything is working as it should be.
+:::
 
 ---
 
-## Helpful getting started videos
+## What is Databricks?
 
-See the collection of [getting started videos](https://educationgovuk.sharepoint.com/:f:/r/sites/ADA-SuperUserForum/Shared%20Documents/General/Getting%20started%20videos?csf=1&web=1&e=RCFSib) covering the basics of Databricks and using notebooks. 
+---
```

Comment:
Line break to remove, sorry!
![image](https://github.com/user-attachments/assets/0e3ff5b5-3ee3-4a0e-a85e-2e9823a2101e)